### DISCUSSION TITLE
Wiki for mobile

### DISCUSSION THREAD

**Discussion by Jerome2103** [community/community#9566] [Mobile]
Why i cant read the github wiki on the mobile app? (Android)

**Comment by brianlovin** [comment]
Hey @Jerome2103 thanks for the note — we don't support the Wiki feature today, and don't have any immediate plans to support it in the near future. But I'll add your request to our tracking issues in case we decide to revisit Wikis in the future.

**Reply by Kris-wag** [reply]
I also request this feature :)

**Reply by JordanWelsman** [reply]
Please also note my request for this feature @brianlovin.
> Jordan

**Reply by 1001daysofcode** [reply]
Same.

**Reply by DitroniX** [reply]
Being able to access wiki pages using the android GitHub app would be so useful.

**Reply by RandomDSdevel** [reply]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;And the same for the iOS GitHub app.

**Reply by Mensh123** [reply]
A lot of projects use the wiki feature to provide guides and reveal technical details. Adding it to mobile would be a huge quality of life improvement as opening github.com in your browser and viewing the wiki from there is a rather complicated solution.

**Reply by Manhunter07** [reply]
The mobile app of GitHub is generally not a very functional implementation, lagging a lot of features including access (at least, readonly) to the wiki.
Since documenting is something I can do when travelling (you dont have power supply in all busses and trains), being able to access the wiki would be enormously useful.
Programming isn't possible on Android (or iOS) anyway, so doing anything like committing/pulling this on a mobile app is pointless.

**Reply by bigchapman** [reply]
I'd like to see this implemented

**Reply by davood-dorostkar** [reply]
it is vital. many repositories are meaningless without accessing their wiki.

**Reply by kod3000** [reply]
It makes no sense that something that has been part of GitHubs inception is left out for so long. Why have this feature at all if you're not going to fully support it.. and development wise we are talking about just making an additional view with links. Making the wiki's readonly on mobile app is acceptable compromise. Not supporting it at all is lazy on part of GitHub and it's entire team.

**Reply by abranczyk** [reply]
I would also really appreciate this feature. I hope you consider revisiting wikis soon. Thank you.

**Reply by ChristopherLyon** [reply]
Bump. This needs to be fast tracked.

**Reply by dgenette** [reply]
+1

**Reply by jlanssie** [reply]
+1

**Reply by mbcann01** [reply]
+1

**Reply by henry-hsieh** [reply]
+1

**Reply by KirillSchmidt** [reply]
+1

**Reply by Dat-Pudding** [reply]
It's been years now. 
Did it at least move up the To-Do ladder in the meantime?

**Reply by NicholasJBond** [reply]
+1

**Reply by IzetopeDev** [reply]
+1! Having a wiki on phone would be immensly useful for working on projects on the go! Hopefully plans to implement a wiki will be expedited!!!

**Reply by Hoff16** [reply]
5 years later and still not even an update...

**Reply by DitroniX** [reply]
This is a major drop-off in GitHub services.  

We have to pay for the 'Wiki feature' and its continued lack of mobile support is inexcusable in todays World.

For such an important platform, used around the World, the GitHub Platform itself is unnecessarily disjointed.

NB. The poor and inconsistent text MD editors throughout the GitHub system does not help either.

**Comment by DitroniX** [comment]
The fact that the Wiki pages are 'promoted' as part of the 'paid for services’, ‘Host documentation and simple websites for your project in a wiki format that contributors can easily edit either on the web or command line', was one of the reasons I upgraded to a paid for package subscription a while back.

Totally agree that the lack of Mobile Wiki support, which is an essential part of the GitHub, is inexcusable on the part of the GitHub development team and the GitHub ‘For-Profit’ Venture Investors.  Make the platform something to be proud of and not rest on one's laurels.

PS.  A more professional Markdown editor, than what is currently provided by GitHub, would not go amiss either….  The free offerings from other like https://stackedit.io/ puts GitHub to shame.

**Comment by freakent** [comment]
I use GitHub on mobile a lot to manage an open source project. I thought someone had deleted my wiki until I read this. It's a major omission from the mobile app for me. At the least, include a link that opens the wiki in my mobile web browser.

**Comment by dfr62** [comment]
I agree that the wiki feature has to be supported on the phone app too!

**Comment by mxchinegod** [comment]
Yeah sorry to invite work but this is a basic part of the ecosystem.

I know it matters to developers a lot which device they have to code for but a reminder: it's because for users it doesn't. They want to use what they want to use.

I have to send communication like this after we've worked to document something well

"sorry i didn’t see this earlier! There is a github wiki for this but I notice the link doesn’t exist on github mobile… so if I send it here it won’t load on your phones browser."

This is pretty obnoxious to explain

**Comment by esheyw** [comment]
Really can't stress enough how useless this makes the app for me. Please implement.

**Comment by philion** [comment]
Just double-checked to confirm that I can access the wiki section from a standard web browser, and I cannot from the GitHub app.

I was excited to standardize on a process that included solid mobile support, but it looks like I must recommend using the reactive webapp version instead of the mobile app. Without access to wiki, the app hides important sections of my repos. Even if linked in the README (to avoid the *total erasure* of the wiki docs), I still get dumped to an external browser to the wiki. Making the app extraneous and confusing overhead.

**Comment by cloudlife-22** [comment]
I am about to start a new org and considering Github enterprise, having a mobile app that allows me to edit wikis is a key requirement for me, lack of this functionality has made this decision way harder than in needs to be, unfortunately I'm looking for alternatives now.

**Comment by lare96** [comment]
Yeah makes no sense that I can view it on a mobile browser but not the app. I can view and edit code but not a simple wiki? Lmao come on guys

**Comment by Vitogbejeaneudes** [comment]
Hello

**Comment by Vitogbejeaneudes** [comment]
Hello

**Comment by TheSkullss** [comment]
Respone #21276

**Reply by freakent** [reply]
> Respone #21276 

This doesn't have anything to do with the Wiki.

**Comment by Vitogbejeaneudes** [comment]
Celà n'a rien avoir avec wiki

Le sam. 8 juin 2024, 14:01, Martin Jarvis ***@***.***> a
écrit :

**Comment by dataspun** [comment]
The fact that github app **_still_** doesn't support github wiki is quite sad  :-(

All I get is 404 but it should be 501

**Comment by Jaakkonen** [comment]
There should at least be a button to open the wiki of particular repository. Now if you have the app installed on Android you cannot even see if a repository has wiki associated with it since all links from browser open in the app.

**Comment by vd2org** [comment]
+1

**Comment by DitroniX** [comment]
Have experienced a number of times now where users use the mobile, and Android app, and then ask me why I do not have a Wiki.  It is actually quite embarrassing, and frustrating, to know that I am spending hours keeping the various Wikis updated and users do not know, or realize, they exist.

As I pay, contribute, to the GitHub service, I do feel even stronger that whoever is holding the ultimate development mantle, continues to exploit and ignore the users basic needs.    Even what is provided is poorly supported, such as the MD Editor etc.

**My question is, why?**

Continue to take regular backups of the both main and wiki gits, with a view to eventually moving away once I get time to investigate the options and migration plan.

**Reply by jdex-terworks** [reply]
Here I am migrating away from the self-hosted because GH manages it all better than I alone ever could - but on this and I'm sure a plethora of issues+1

Wiki should be a first-class citizen on mobile, and it probably should have been from their first mobile iteration. Having your docs in the palm of your hand is far more sensible than having the code in your hand in most situations.

**Comment by C0C0B01** [comment]
+1

**Comment by 1Thef0x** [comment]
As of now, the GitHub mobile app (Android) does not support viewing Wiki pages directly within the app. The app is primarily designed for:

Reviewing and merging pull requests

Managing issues and discussions

Browsing code and commits

Receiving notifications

there is work around like 
 use  browser version

**Comment by DitroniX** [comment]
The browser view on a mobile is of much use as a chocolate crayon when it comes to a new user looking for Wiki information.  

It certainly is not an ideal workaround for a professional collaboration tool where users pay for a Git service which is only partly and inconsistently supported.

**Comment by spenserblack** [comment]
I regularly answer issues that have their solutions documented in the wiki. When I read those issues on mobile, my first instinct is to go to the wiki to confirm the answer I'm about to give... but there is no wiki on mobile. Having the wiki available on mobile is very useful for being able to have a source for your issue and discussion answers, and not being able to easily see the wiki can hinder conversations.

Thankfully, in my particular instance, the wiki was *also* tracked in the main repo's code (we sync the wiki content with content in a `wiki/` folder), but if we hadn't set that up then this would have been pretty annoying for me.

**Comment by alimdabbas** [comment]
> Really can't stress enough how useless this makes the app for me. Please implement. 

Game hacking
